neverbleed.c

```diff
@@ -450,7 +497,9 @@ static struct {
         struct key_slots ecdsa_slots;
     } keys;
     neverbleed_t *nb;
-} daemon_vars = {{PTHREAD_MUTEX_INITIALIZER}};
+} daemon_vars = {
+    .keys.lock = PTHREAD_MUTEX_INITIALIZER,
+};
```

Comment:
This change is unnecessary, correct?

I would not mind start using named initialization here, though I'm not sure if nesting them (`.keys.lock`) is C99.

Reply:
That's correct, it's unnecessary.

FWIW:
It's `designated initializer` in the C99 (https://gcc.gnu.org/onlinedocs/gcc/Designated-Inits.html)